src/VirtoCommerce.NotificationsModule.Data/Services/NotificationService.cs

```diff
@@ -120,6 +122,23 @@ private void ClearCache(Notification[] notifications)
             }
         }
 
+        private void SetTransient(NotificationEntity entity)
```

Comment:
Is this code no longer relevant, or you decide to do not do this on save changes?

Reply:
add reset entity data to NotificationEntity